src/FieldLabel/Outlined/index.js

```diff
@@ -20,58 +19,42 @@ const styles = css`
         cursor: not-allowed;
     }
 
-    .dense {
-        height: ${inputHeightDense + 10}px;
+    .content {
+        align-items: center;
+        box-sizing: border-box;
+        display: flex;
+        height: 42px;
+        left: 1px;
+        position: relative;
+        width: calc(100% - 1px);
```

Comment:
I'm not ecstatic about these hacks, can we put a comment about what the situation is that requires it so we know when they can be removed? :thinking: